client/app.jsx

```diff
@@ -1,8 +1,77 @@
 import React from 'react';
 import Home from './pages/home';
+import Navbar from './navbar'
+import axios from 'axios';
 
 export default class App extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      games: []
+    };
+  }
+
+  componentDidMount() {
+    axios.get('http://localhost:3001/getGames')
```

Comment:
All of these `localhost:3001` are going to break when you deploy your app. Just use `/getGames`